managed/services/telemetry/config.default.yml

```diff
@@ -108,6 +108,62 @@ telemetry:
       - metric_name: "mongodb_version"
         label: "mongodb"
 
+  - id: MongoDBExporterMetricsGeneralStats
+    source: VM
+    query: collector_scrape_time_ms{exporter="mongodb",collector="general"}
+    summary: "Meta metrics for MongoDB exporter"
```

Comment:
can you please provide a bit more detailed description, this will be used in the UI, so the users will be seeing

```
Meta metrics for MongoDB exporter
Meta metrics for MongoDB exporter
Meta metrics for MongoDB exporter
Meta metrics for MongoDB exporter
Meta metrics for MongoDB exporter
...
```

which will look as a bug

Reply:
Done